Workade/Views&ViewModels/MyPage/ProfileView.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  ProfileView.swift
+//  Workade
+//
+//  Created by Hong jeongmin on 2022/11/16.
+//
+
+import UIKit
+
+class ProfileView: UIView {
+    // TODO: Login User 정보 ViewModel 사용
+    
+    let containerView: UIView = {
+        let containerView = UIView()
+        containerView.backgroundColor = .clear
+        containerView.translatesAutoresizingMaskIntoConstraints = false
+        
+        return containerView
+    }()
+    
+    private let profileImage: UIImageView = {
+        let imageView = UIImageView()
+        imageView.layer.cornerRadius = 32
+        imageView.backgroundColor = .theme.labelBackground
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        
+        // TODO: 로그인 유저 프로필사진 정보로 변경
+        
+        return imageView
+    }()
+    
+    private lazy var editProfileButton: UIButton = {
+        let button = UIButton()
+        button.setTitle("프로필 편집", for: .normal)
+        button.setTitleColor(.theme.tertiary, for: .normal)
+        // TODO: caption2 2차 디자인 시스템 적용
+        button.titleLabel?.font = .customFont(for: .caption)
+        button.backgroundColor = .theme.labelBackground
```

Comment:
```suggestion
        button.backgroundColor = .theme.groupedBackground
```
[P2] 로 바꿔주시면 감사하겠습니다!

Reply:
반영 했습니다!